LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -1119,30 +1094,35 @@
             type: 'GET',
             url: '../api/workflow/userActions',
             success: function(res) {
-                let buffer =
-                    `<table id="actions" class="table" border="1">
-                        <caption><h2>List of Actions</h2></caption>
-                        <thead>
-                            <th scope="col">Action</th>
-                            <th scope="col">Action (Past Tense)</th>
-                            <th scope="col">Button Order</th>
-                            <th scope="col"></th>
-                        </thead>`;
+                let buffer = `<table id="actions" class="table" border="1">
+                    <caption><h2>List of Actions</h2></caption>
+                    <thead>
+                        <th scope="col">Action</th>
+                        <th scope="col">Action (Past Tense)</th>
```

Comment:
I have concerns this _may_ cause confusion.  Don't make changes now.  Just try to remember demo to point this out and ask if they want changes.  This would be handled on a _new_ ticket, not this one.  Might what to get the 508 team to look at it.